app/konnect/getting-started/publish-service.md

```diff
@@ -46,27 +46,28 @@ descriptions.
 
     ```
 
-1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), open a service.
-Find the **Service Document** section on your service's overview page and click **Upload Document**.
+1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service.
+
+1. Select **Documentation**, then **Add Page**. Upload the `.md` file and **Save**.
```

Comment:
```suggestion
1. Select **Documentation**, then **Add Page**. Upload the `.md` file and click **Save**.
```
Optional, I think it makes sense either way. I'm just used to doing "click" before **Save**.